advisories/published/2025/JLSEC-0000-mntg6n4tg-1shrccz.md

```diff
@@ -0,0 +1,24 @@
+```toml
+schema_version = "1.7.4"
+id = "JLSEC-0000-mntg6n4tg-1shrccz"
+modified = 2025-11-20T23:04:01.828Z
+upstream = ["CVE-2021-43666"]
+references = ["https://github.com/ARMmbed/mbedtls/issues/5136", "https://lists.debian.org/debian-lts-announce/2022/12/msg00036.html", "https://github.com/ARMmbed/mbedtls/issues/5136", "https://lists.debian.org/debian-lts-announce/2022/12/msg00036.html", "https://lists.debian.org/debian-lts-announce/2025/06/msg00034.html"]
+
+[[affected]]
+pkg = "MbedTLS_jll"
+ranges = ["*"]
```

Comment:
This was back ported to 2.x in https://github.com/Mbed-TLS/mbedtls/commit/384b98bdaed86febd179f6479f43ed4a5cda8502, which landed in 2.28.0

```suggestion
ranges = ["< 2.28.0+0"]
```